src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", Arial, sans-serif;
+}
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
```

Comment:
redundant

Reply:
Cannot remove dispaly:flex; and justify-content: space-between because it is change layout